migration to bootstrap 2.0

theJenix opened the pull request:
this pull request contains the code to migrate the site from bootstrap 1.4 to bootstrap 2.0

theJenix committed:
Revert "first cut at markdown preview functionality"

This reverts commit 3f464b93fe4e4f26ba047d31051e20fde3bb3bdd.

theJenix committed:
Merge branch 'master' of git://github.com/cc-wit/attendance

theJenix committed:
Merge branch 'master' of git://github.com/cc-wit/attendance

theJenix committed:
migrate to bootstrap 2.0

first pass at migration to bootstrap 2.0

theJenix committed:
login form (main/collapsed)

fixed login form look and feel
set up login form in collapsed menu to be 1st thing in the list.

theJenix committed:
drop downs and collapsable nab bar

fixed a bug where the drop down menu would not show correctly after
collapsing and uncollapsing the nav bar

theJenix committed:
migrated forms

migrated forms to use new bootstrap2 classes

theJenix committed:
replaced express-bootstrap-messages

replaced with a lib of our own, based on the express-bootstrap-messages
but updated to use bootstrap2.0 classes

theJenix committed:
migrated recovery form

migrated to bootstrap 2.0

thelastnode commented on lib/bootstrap2-messages.js:
```diff
@@ -0,0 +1,28 @@
+module.exports = function(req, res) {
+  return function() {
+    var buf, i, j, len, messages, msg, msgs, type, types, _ref;
```
Just a style nit, but can we move these declarations to where they're defined? It'll be more consistent with the rest of the code, and we can always "use strict" if it's a problem

theJenix committed:
commented the copy n paste

added a comment noting that this code was copied and pasted from
express-messages-bootstrap (with some minor changes)

theJenix committed:
changed to use minified bootstrap files

changed the layout to use the .min versions of the css and js files

thelastnode commented:
In `events.jade`, in the footer (lines 20 and 23), since Bootstrap 2.0 is a 12 column grid instead of a 16 column grid, use `.span2` and `.span2.offset8` as the CSS classes

theJenix committed:
changed class to fit within bootstrap 2.0 grid

changed .offset12 to .offset8, to fit within bootstrap2.0 12 column grid

thelastnode commented:
This is awesome.
